src/client/components/NameInputField.tsx

```diff
@@ -83,6 +84,7 @@ const SecondNameInput = (props: NameInputProps) => {
 			onInvalid={onInvalid}
 			error={errorMessage}
 			defaultValue={props.defaultValue}
+			theme={textInputTheme}
```

Comment:
I notice this pattern is everywhere.. would it be worth making a new component called `ThemedTextInput` which automatically applies the theme to the source `TextInput` component. And then use the `ThemedTextInput` everywhere the `TextInput` was being used.

This would mean that we wouldn't have to worry about future developers failing to add the `theme` prop everywhere.

Reply:
Yeah, that makes sense! Like for the links.